plugins/reposync.py

```diff
@@ -181,25 +199,22 @@ def get_pkglist(self, repo):
             query.filterm(arch=self.opts.arches)
         return query
 
-    def download_packages(self, repo):
+    def download_packages(self, repo, pkglist):
```

Comment:
The ``repo`` argument is no longer used; it could be removed, too.

Reply:
Thanks, removed.